tool/tsh/db.go

```diff
@@ -275,7 +292,7 @@ func onDatabaseConfig(cf *CLIConf) error {
 	if err != nil {
 		return trace.Wrap(err)
 	}
-	profile, err := client.StatusCurrent(cf.HomePath, cf.Proxy)
+	profile, err := client.StatusCurrentWithIdentity(cf.HomePath, cf.Proxy, cf.IdentityFileIn)
```

Comment:
Seeing how many instances of `StatusCurrent` had to be updated I wonder if we can have a helper method taking in CLIConf instead? This way we can ensure consistency across the board and reduce the risk of someone using the wrong method (without `cf.IdentityFileIn`)

Reply:
I've gone ahead and merged all the functionality into `StatusCurrent()`. We need to pass along the identity path but it's easily available everywhere, so I think this should be modestly nicer overall.